person_capture/gui_app.py

```
@@ -3801,15 +3806,19 @@ def _pipe_like(obj) -> bool:
 
             # Default: HDR passthrough ON when available unless explicitly disabled in cfg.
             cfg_hdr_passthrough = bool(getattr(cfg, "hdr_passthrough", True))
+            prescan_will_run = bool(getattr(cfg, "prescan_enable", True)) and total_frames > 0
```

Ruff (0.15.11)

[error] Undefined name `total_frames`

(F821)